src/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,107 @@
+import "./filter.css";
+
+function Filter() {
+  return (
+    <div class="productlisting_sidebar">
+      <div class="d-flex products_filter_section">
+        <h2>Filters</h2>
+        <button class="btn custom_clear_btn">Clear</button>
+      </div>
+      <div class="mb-2">
+        <h4 class="products_sidebar_title">Category</h4>
+        <div class="products_sidebar_inputbox">
+          <input type="checkbox" />
+          <label class="ml-1" for="">
```

Comment:
either remove for attribute or give it a proper value

Reply:
I will be  using it in future , If I dont I will remove it